ODIN_II/SRC/include/odin_error.h

```diff
@@ -28,6 +28,8 @@ enum odin_error {
     PARSE_BLIF,
     /* for errors in the netlist simulation */
     SIMULATION,
+    /* for incomplete implementation error messages*/
+    INC_IMPLEMENTATION,
```

Comment:
the errors are for phases only, since this is in partial mapping, make sure to use the right one (NETLIST in your case)

Reply:
Thank you, I will account with that suggestion!